addons/mail/i18n/mail.pot

```diff
@@ -443,6 +443,11 @@ msgstr ""
 msgid "<strong>Recommended Activities</strong>"
 msgstr ""
 
+#. module: mail
+#: code:addons/mail/models/mail_channel.py:0
```

Comment:
I'm not sure the importance of these comments, but they look different than others. In particular it looks like it is missing `#, python-format`

Reply:
the `python-format` flag is for string which need to be formatted using python afterward (https://www.gnu.org/software/gettext/manual/html_node/python_002dformat.html#python_002dformat)

As for the `#:`  it serves as reference to indicate where it's used -> here in the code of the given file
I guess the `model:` start allow to know it's defined in a model statement (usually the fields) but i'm not sure.